core/src/test/java/org/hibernate/ogm/backendtck/storedprocedures/Car.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.storedprocedures;
+
+import static org.hibernate.ogm.backendtck.storedprocedures.indexed.IndexedStoredProcedureCallTest.TEST_RESULT_SET_STORED_PROC;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.EntityResult;
+import javax.persistence.Id;
+import javax.persistence.NamedStoredProcedureQueries;
+import javax.persistence.NamedStoredProcedureQuery;
+import javax.persistence.ParameterMode;
+import javax.persistence.SqlResultSetMapping;
+import javax.persistence.StoredProcedureParameter;
+
+/**
+ * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
+ */
+@Entity
+@NamedStoredProcedureQueries({
+		@NamedStoredProcedureQuery(name = "testproc4_1", procedureName = TEST_RESULT_SET_STORED_PROC, parameters = {
+				@StoredProcedureParameter(mode = ParameterMode.REF_CURSOR, type = Void.class),
+				@StoredProcedureParameter(mode = ParameterMode.IN, type = String.class),
+				@StoredProcedureParameter(mode = ParameterMode.IN, type = String.class)
+		}, resultClasses = Car.class),
+		@NamedStoredProcedureQuery(name = "testproc4_2", procedureName = TEST_RESULT_SET_STORED_PROC, parameters = {
+				@StoredProcedureParameter(mode = ParameterMode.REF_CURSOR, type = Void.class),
+				@StoredProcedureParameter(mode = ParameterMode.IN, type = String.class),
+				@StoredProcedureParameter(mode = ParameterMode.IN, type = String.class)
+		}, resultSetMappings = "carMapping")
+})
+
```

Comment:
Empty line :)
More important, it seems you are only testing for `String` type, you should also check that it works for Integer and Double parameters

Reply:
Nice test overall, though

Reply:
It might also be helpful to test a storedprocedure that does not require parameters

Reply:
Okey. I change type of someone field.